CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [[3.1dev](https://github.com/nf-core/rnaseq/releases/tag/3.1)] - 2021-03-26
```

Comment:
I wasn't clear on why the dev version in the changelog already had a tag and a release date..? So I deleted it 👀